src/lib/components/link.svelte

```diff
@@ -6,13 +6,12 @@
 
 <style>
   .link {
-    display: flex;
     text-decoration: none;
     font-weight: bold;
     color: var(--interactive-button);
     position: relative;
-    justify-content: space-between;
     z-index: 1;
+    gap: 10px;
```

Comment:
gap only works with flex

Reply:
It seems that flex is no longer needed because if you separate the content from the link

Reply:
exactly, if it's not needed we delete it